SwiftPackage.xcodeproj/project.pbxproj

```diff
@@ -931,7 +931,6 @@
 				PRODUCT_NAME = SwiftPackage;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
I usually use `default`.

Reply:
Yeah, it will defaults to the project setting, which is also default